infrastructure/app/stages/prod.yaml

```diff
@@ -1,6 +1,13 @@
 service:
   type: NodePort
 ingress:
+  probe:
+    enabled: true
+    targets:
+      staticConfig:
+        static:
+          - https://www.citizensadvice.org.uk/consumer/energy/energy-supply/save-energy-at-home/check-how-much-your-electrical-appliances-cost-to-use/
+          - https://www.citizensadvice.org.uk/consumer/your-energy/get-a-better-energy-deal/compare-domestic-energy-suppliers-customer-service/
```

Comment:
Using the full urls for the main pages for each app here. There didn't seem to be a nice equivalent version of the shorter qa url.